airflow/dag_processing/manager.py

```diff
@@ -695,15 +696,16 @@ def _fetch_callbacks_with_retries(self, max_callbacks: int, session: Session):
         """Fetch callbacks from database and add them to the internal queue for execution."""
         self.log.debug("Fetching callbacks from the database.")
         with prohibit_commit(session) as guard:
-            query = session.query(DbCallbackRequest)
+            query = select(DbCallbackRequest)
             if self.standalone_dag_processor:
-                query = query.filter(
+                query = query.where(
                     DbCallbackRequest.processor_subdir == self.get_dag_directory(),
                 )
             query = query.order_by(DbCallbackRequest.priority_weight.asc()).limit(max_callbacks)
             callbacks = with_row_locks(
                 query, of=DbCallbackRequest, session=session, **skip_locked(session=session)
-            ).all()
+            )
+            callbacks = session.scalars(callbacks)
```

Comment:
I have a feeling Mypy might not like this at some point when SQLAlchemy adds typing support. Do this instead:

```python
query = with_row_locks(...)
callbacks = session.scalars(query)
```

Reply:
done

Reply:
changed it